<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование путей и правил</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushButton_add_out">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../Downloads/icons8-плюс-50.png</normaloff>../../../../Downloads/icons8-плюс-50.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QListWidget" name="listWidget_filters">
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="pushButton_del_out">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../Downloads/icons8-минус-64.png</normaloff>../../../../Downloads/icons8-минус-64.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButton_add_rule">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../Downloads/icons8-плюс-50.png</normaloff>../../../../Downloads/icons8-плюс-50.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QListWidget" name="listWidget_out">
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_add_filter">
     <property name="maximumSize">
      <size>
       <width>41</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../Downloads/icons8-плюс-50.png</normaloff>../../../../Downloads/icons8-плюс-50.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QListWidget" name="listWidget_rules"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_add_in">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../Downloads/icons8-плюс-50.png</normaloff>../../../../Downloads/icons8-плюс-50.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget_in">
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButton_del_filter">
     <property name="maximumSize">
      <size>
       <width>41</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../Downloads/icons8-минус-64.png</normaloff>../../../../Downloads/icons8-минус-64.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Список фильтров</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Список папок на подпись</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="6">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="pushButton_del_rule">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../Downloads/icons8-минус-64.png</normaloff>../../../../Downloads/icons8-минус-64.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Список назначений</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_del_in">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../Downloads/icons8-минус-64.png</normaloff>../../../../Downloads/icons8-минус-64.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Список созданных правил (Не распространяются на документы, направленные через Мини ЭДО)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
